server/protocols/jmap/src/main/java/org/apache/james/jmap/model/Message.java

```diff
@@ -163,7 +163,7 @@ public Builder date(Instant date) {
         }
 
         public Builder size(long size) {
-            this.size = size;
+            this.size = Number.lenientFactory().fromLong(size);
```

Comment:
why lenient here?

Reply:
It's from our code -> to the user. As far as I know we never parse Messages (but CreationMessage).

If we don't want to fail badly on negative size (for instance) we have to use a lenient stuff.

Reply:
then Number.fromOutboundLong(size) or something like that?